File: java/picocontainer/src/test/org/picocontainer/defaults/DefaultMultipleConstructorTestCase.java
package org.picocontainer.defaults;

import org.picocontainer.tck.AbstractMultipleConstructorTestCase;
import org.picocontainer.MutablePicoContainer;

/**
 * @author Aslak Helles&oslash;y
 * @version $Revision$
 */
public class DefaultMultipleConstructorTestCase extends AbstractMultipleConstructorTestCase {
    protected MutablePicoContainer createPicoContainer() {
        return new DefaultPicoContainer();
    }
}

File: dotnet/picocontainer/PicoContainer.Test/PicoContainer.Defaults/DefaultMultipleConstructorTestCase.cs
using NUnit.Framework;
using PicoContainer.Core;
using PicoContainer.Defaults;
using PicoContainer.Tck;

namespace PicoContainer.Defaults
{
	[TestFixture]
	public class DefaultMultipleConstructorTestCase : AbstractMultipleConstructorTestCase
	{
		protected override IMutablePicoContainer createPicoContainer()
		{
			return new DefaultPicoContainer();
		}
	}
}